MdeModulePkg/Bus/Pci/CxlDxe/CxlDxe.inf

```diff
@@ -0,0 +1,40 @@
+## @file
+#
+#  CxlDxe driver setup file
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+[Defines]
+  INF_VERSION                    = 0x0001001B
+  BASE_NAME                      = CxlDxe
+  FILE_GUID                      = 29e54f5e-365a-4235-ba62-c832d7660852
+  MODULE_TYPE                    = DXE_DRIVER
+  VERSION_STRING                 = 1.0
+  ENTRY_POINT                    = CxlDxeEntryPoint
+
+[Sources.common]
+  CxlDxe.c
+  CxlDxe.h
+  CxlDxe_fw_mgmt.c
+  CxlDxe_util.c
+
+[Packages]
+  EmbeddedPkg/EmbeddedPkg.dec
```

Comment:
Dose this module depend on EmbeddedPkg？

Reply:
Review comment is taken care in updated patch. EmbeddedPkg is removed.